function handleError(){
    console.log('This is used to handle errors');
}